ngraph/core/shape_inference/include/convolution_shape_inference.hpp

```diff
@@ -0,0 +1,172 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <openvino/op/convolution.hpp>
+
+
+namespace ov {
+namespace op {
+namespace v1 {
+
+
+template<class T>
+void shape_infer(Convolution* op, const std::vector<T> &input_shapes, std::vector<T> &output_shapes) {
```

Comment:
Maybe convolution_shape_infer?

Reply:
It consumes Convolution node, isn't it enough?